controllers/instanaagent_controller.go

```diff
@@ -43,11 +51,16 @@ const (
 	crExpectedNamespace   = "instana-agent"
 )
 
+var (
+	minimumSupportedChartVersion, _ = semver.Parse("1.2.28")
```

Comment:
While the Helm charts ignore unknown configuration options, we want to prevent customers picking too old versions.
Starting just with the current version, which should be a nice baseline also including all features (such as the recently added TLS support).